### DISCUSSION TITLE
KanboardCSS v3.5

### DISCUSSION THREAD

**Discussion by aljawaid** [aljawaid/KanboardCSS#21] [Announcements]
## 3.5

_(most recent changes are listed on top):_
- 3778c95 NEW: Add Code Scanning Badge to README.md
- 7fd80e6 FIX: Code Scanning: CSS Fixes
- 770e4d0 FIX: Code Scanning: Avoid Warnings
- e3b4b55 FIX: Code Scanning: Delete Duplicates
- fd87d6a FIX: Code Scanning: Delete Duplicates
- 70376d6 FIX: Code Scanning: Delete Duplicates
- 6891112 FIX: Code Scanning: CSS Fixes
- f0506c1 FIX: Code Scanning: CSS Fixes
- 0d2a6cb FIX: Code Scanning: CSS Fixes
- cbaac95 FIX: Code Scanning: CSS Fixes
- b39d2a0 FIX: Code Scanning: CSS Fixes
- 1c100a9 FIX: Code Scanning: Major CSS Fixes
- caa408c FIX: README Navigation Links for GitHub
- a58df52 FIX: Code Scanning: Expected empty line before rule
- 677b37f FIX: Code Scanning: Unexpected Unit
- affddec FIX: Move Archived Releases to `Releases`
- 35afd0c FIX: Code Syntax - Unexpected leading zero
- a6e2baa FIX: Code Syntax in Translations
- 4c5df39 FIX: Code Syntax in changelog.md
- 02877b2 FIX: Anchor Link Position
- 70eb847 FIX: Code Syntax
- 3533d6c FIX: changelog.md
- 29c74ee NEW: Add Footer Badges to README.md
- 64a7094 NEW: Add Section Navigation Links to README.md
- 3292b63 NEW: Add Badges to README.md
- 719b92c FIX: Add README Page Navigation Links
- 43b64fb NEW: Add README Page Navigation Links
- 4d7576d FIX: Content Flow for README.md
- 5baa6b7 NEW: Create INSTALL.md
- 84e5f32 NEW: Add Download Badge to README.md
- 6575fe1 NEW: Stronger Alert Notification - also delay timeout to 10s instead of default 7s
- 1da5284 NEW: Move Alerts to The Top of Screen
- d9d997d FIX: Task Sidebar Font Size
- 9d1e31a FIX: Sidebar Indicator & Alignment
- f6eebba FIX: Sidebar Font Weight
- c42eed4 FIX: Sidebar was too restrictive
- b19d695 FIX: Alert - fixed alerts were auto-fading too
- 6e6a173 FIX: Remove Duplicate Code

---

Read the full [**Changelog**](../master/changelog.md "See changes") or view the [**README**](../master/README.md "View README")

<hr /><em>This discussion was created from the release <a href='https://github.com/aljawaid/KanboardCSS/releases/tag/v3.5'>KanboardCSS v3.5</a>.</em>

**Comment by aljawaid** [comment]
# Recommended Release

- A major cleanup of code, syntax and errors was done in this version.

**Reply by mokamobile** [reply]
HI. Sorry for a stupid question, but is there a chance of keeping the V1.5 design for latest version? new design is kind of heavy...

**Reply by aljawaid** [reply]
Hello, I'm afraid v1.5 was back in 2018 so reverting to that design would be very complicated for me and somewhat timeconsuming.

I will however, work on making the design lighter over the next few releases.

**Reply by aljawaid** [reply]
_Kindly 'star' the project or folow me for updates_

**Reply by mokamobile** [reply]
Thank you for your work
